template/src/project_name/main/settings.py

```diff
@@ -144,3 +146,18 @@
         },
     },
 }
+
+# Sentry
+sentry_sdk.init(
+    dsn=env("SENTRY_DSN", default=None),
+    # Set traces_sample_rate to 1.0 to capture 100%
+    # of transactions for tracing.
+    traces_sample_rate=env.float("SENTRY_TRACES_SAMPLE_RATE", default=1.0),
+    # Set profiles_sample_rate to 1.0 to profile 100%
+    # of sampled transactions.
+    # We recommend adjusting this value in production.
+    profiles_sample_rate=env.float("SENTRY_PROFILES_SAMPLE_RATE", default=1.0),
```

Comment:
issue: sampling all transactions is way too much and will chew through quota very quickly

We typically have this default to 0 because unlike error monitoring this kind of performance monitoring tends to be a lot less defined in terms of support coverage while being a lot more frequent in terms of quota spend - having it at 0 by default means we can safely not worry that a hearty application will eat up our quota unexpected while still letting us opt-in in a per app basis as needed via the env variable

```suggestion
    traces_sample_rate=env.float("SENTRY_TRACES_SAMPLE_RATE", default=0.0),
    profiles_sample_rate=env.float("SENTRY_PROFILES_SAMPLE_RATE", default=0.0),
```